src/fsharp/FSharp.Core.Unittests/FSharp.Core/Microsoft.FSharp.Control/AsyncModule.fs

```diff
@@ -6,8 +6,126 @@
 namespace FSharp.Core.Unittests.FSharp_Core.Microsoft_FSharp_Control
 
 open System
+open System.Threading
 open FSharp.Core.Unittests.LibraryTestFx
 open NUnit.Framework
+#if !(FSHARP_CORE_PORTABLE || FSHARP_CORE_NETCORE_PORTABLE)
```

Comment:
An easy solution for now would be to incorporate the non-FsCheck tests from #744